### ISSUE TITLE
can't export

### ISSUE THREAD

**Issue by Little-Star-Star** [mtxr/vscode-sqltools#868]
Issue Type: <b>Bug</b>

can't export

Extension version: 0.23.0
VS Code version: Code 1.60.1 (83bd43bc519d15e50c4272c6cf5c1479df196a4d, 2021-09-10T17:07:10.714Z)
OS version: Windows_NT x64 10.0.18363
Restricted Mode: No
Remote OS version: Linux x64 4.15.0-142-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 3500U with Radeon Vega Mobile Gfx   (8 x 2096)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|6.94GB (3.76GB free)|
|Process Argv|--crash-reporter-id e652c2bf-c266-491d-a5fe-96028ce2c2df|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: 121.5.203.35|
|OS|Linux x64 4.15.0-142-generic|
|CPUs|Intel(R) Xeon(R) Gold 6133 CPU @ 2.50GHz (1 x 2494)|
|Memory (System)|1.79GB (0.10GB free)|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspyt653:30270858
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551:30345470
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
pythonvsuse255:30340121
vscod805:30301674
pythonvspyt200:30340761
vscextlang:30333561
binariesv615:30325510
vsccppwtct:30364498
pythonvssor306:30344512
bridge0708:30335490
pygetstartedt2:30360495
bridge0723:30353136
pythonrunftest32:30365366
pythonf5test824:30361777
javagetstartedc:30364665
pythonvspyt187cf:30365362
pydsgst2:30361792
vssid140cf:30363604
vssur157:30367808

```

</details>

**Comment by Little-Star-Star** [comment]
can't export

**Comment by gjsjohnmurray** [comment]
If you still have this problem please provide more information.

**Comment by auduny** [comment]
I have the same problem, same version of extension, but running on OS X on M1 arch. running agains a mysqldatabase

I get a 

`Request connection/SaveResultsRequest failed with message: Cannot destructure property 'results' of '(intermediate value)' as it is undefined.`

When i try to export or open to cvs or json

It seems to be related to issue #832 with pull-request https://github.com/mtxr/vscode-sqltools/pull/880

**Comment by przemek-oc** [comment]
+1 
`[2022-08-24 17:12:23.870] [exthost] [error] Error: Request connection/SaveResultsRequest failed with message: Cannot destructure property 'results' of '(intermediate value)' as it is undefined.
	at c:\Users\username\.vscode\extensions\mtxr.sqltools-0.24.0\dist\extension.js:2:114042
	at c:\Users\username\.vscode\extensions\mtxr.sqltools-0.24.0\dist\extension.js:2:114449
	at Immediate.<anonymous> (c:\Users\username\.vscode\extensions\mtxr.sqltools-0.24.0\dist\extension.js:2:114991)
	at processImmediate (node:internal/timers:464:21) sqltools.saveResults {"value":"mtxr.sqltools","_lower":"mtxr.sqltools"}`

Works well after downgrade extension to 0.23

**Comment by gjsjohnmurray** [comment]
Should be fixed in today's releases (0.25.0 main extension, 0.4.0 MySQL and SQLite drivers). If any of you still get these issues after upgrading please report here.
